### DISCUSSION TITLE
Info logs not printed though Log::SetVerbosity(Log::Kind::Info)

### DISCUSSION THREAD

**Discussion by botoph4** [eProsima/Fast-DDS#3504] [Support]
### Is there an already existing issue for this?

- [X] I have searched the existing issues

### Expected behavior

Print info and warn level as well as error

### Current behavior

only print error.

```
➜  build ./DDSHelloWorldPublisher
2023-05-09 11:53:21.124 [P_INFO Warning] Warn. -> Function main                                                                                      
2023-05-09 11:53:21.124 [P_INFO Error] Error. -> Function main
```

### Steps to reproduce

```
int main(int argc, char **argv) {
  Log::SetVerbosity(Log::Kind::Info);
  EPROSIMA_LOG_INFO(P_INFO, "Info.");
  EPROSIMA_LOG_WARNING(P_INFO, "Warn.");
  EPROSIMA_LOG_ERROR(P_INFO, "Error.");
```

### Fast DDS version/commit

2.10.1 Master

### Platform/Architecture

Other. Please specify in Additional context section.

### Transport layer

Default configuration, UDPv4 & SHM

### Additional context

Ubuntu 22.04

**Comment by JLBuenoLopez** [comment]
Hi @botoph4 

For performance sake, info logs are not build in the library unless explicitly enabled. Please, read this section of [Fast DDS documentation](https://fast-dds.docs.eprosima.com/en/latest/fastdds/logging/disable_module.html#disable-logging-module).

What installation are you using? Building from sources or directly installing the binaries? If building from sources, which are the CMake options you are setting? I suggest you set the following C++ macros:

```
#define HAVE_LOG_NO_INFO 0
#define FASTDDS_ENFORCE_LOG_INFO 1
```

Finally, as this is not a proper bug but a misconfiguration, I am going to move it to the corresponding discussion forum according with [Fast DDS Contributing guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md#issues-and-support).
